src/um/secext.rs

```diff
@@ -0,0 +1,59 @@
+// Licensed under the Apache License, Version 2.0
```

Comment:
This file should live in `shared/`, as the equivalent header in windows sdk is `Include/shared/secext.h`